rest-client-base/src/main/java/com/wultra/core/rest/client/base/DefaultRestClient.java

```diff
@@ -204,6 +212,50 @@ private void initializeWebClient() throws RestClientException {
         webClient = builder.baseUrl(config.getBaseUrl()).clientConnector(connector).build();
     }
 
+    private static HttpClient configureKeepAlive(final HttpClient httpClient, final RestClientConfiguration config) throws RestClientException {
+        final Duration keepAliveIdle = config.getKeepAliveIdle();
+        final Duration keepAliveInterval = config.getKeepAliveInterval();
+        final Integer keepAliveCount = config.getKeepAliveCount();
+        logger.info("Configuring Keep-Alive, idle={}, interval={}, count={}", keepAliveIdle, keepAliveInterval, keepAliveCount);
+        if (keepAliveIdle == null || keepAliveInterval == null || keepAliveCount == null) {
+            throw new RestClientException("All Keep-Alive properties must be specified.");
+        }
+
+        final int keepIdleSeconds = Math.toIntExact(keepAliveIdle.toSeconds());
+        final int keepIntervalSeconds = Math.toIntExact(keepAliveInterval.toSeconds());
+
+        return httpClient.option(ChannelOption.SO_KEEPALIVE, true)
```

Comment:
It would be nicer to set NIO options or EPOLL options in case this implementation is present.

Reply:
> By default Reactor Netty will use Epoll/Kqueue if the native libraries are on the path, otherwise NIO. You can use runOn configuration to configure this behaviour by yourself

https://stackoverflow.com/questions/72969831/what-does-transport-really-mean-in-webclient-of-spring5-using-reactor-netty#comment129013442_72969831

I would rather configure both than mis-detect the transport layer.